client/src/components/UltimatePaginationTopDown.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from "prop-types"
+import React from "react"
+import UltimatePagination from "components/UltimatePagination"
+
+const UltimatePaginationTopDown = props => {
+  const {
+    componentClassName,
+    className,
+    pageNum,
+    pageSize,
+    totalCount,
+    goToPage,
+    contentElement
+  } = props
+
+  return (
+    <>
+      <UltimatePagination
+        Component={"header"}
+        componentClassName={componentClassName}
+        className={className}
+        pageNum={pageNum}
+        pageSize={pageSize}
+        totalCount={totalCount}
+        goToPage={goToPage}
+      />
+      {contentElement}
+      <UltimatePagination
+        Component={"footer"}
+        componentClassName={componentClassName}
+        className={className}
+        pageNum={pageNum}
+        pageSize={pageSize}
+        totalCount={totalCount}
+        goToPage={goToPage}
+      />
+    </>
+  )
+}
+
+UltimatePaginationTopDown.propTypes = {
+  componentClassName: PropTypes.string,
+  className: PropTypes.string,
+  pageNum: PropTypes.number,
+  pageSize: PropTypes.number,
+  totalCount: PropTypes.number,
+  goToPage: PropTypes.func,
+  contentElement: PropTypes.object
```

Comment:
Strictly speaking, this should be `PropTypes.oneOfType([PropTypes.func, PropTypes.object])` since React elements can also be (pure) functions.

Reply:
Ha, nice one.

Reply:
Indeed. But instead of passing a specific prop, why not just use the standard `props.children` and pass the content element as a child of `UltimatePaginationTopDown`?

Reply:
Agreed, that is a cleaner solution